README.md

```diff
@@ -1,70 +1,100 @@
+The Genesis node is configured via the [materials/signet-genesis-node.sh](https://github.com/stratum-mining/sv2-workshop/blob/main/materials/signet-genesis-node.sh).
 
-The `signet_genesis_node.sh` is responsible for:
-- Deploy a local signet.
-- Mine 16 blocks as bootstrapping for the SRI pool.
+This script:
+* Deploys a local signet using the [`bitcoin.conf`](https://github.com/stratum-mining/sv2-workshop/blob/main/materials/genesis-bitcoin.conf).
+* Mines 16 blocks as bootstrapping for the SRI pool.
 
-In order to use this script, you should first export some environment variables. For example:
-```
-$ export BITCOIND=$HOME/bitcoin/src/bitcoind
-$ export BITCOIN_CLI=$HOME/bitcoin/src/bitcoin-cli
-$ export BITCOIN_UTIL=$HOME/bitcoin/src/bitcoin-util
-$ export MINER=$HOME/bitcoin/contrib/signet/miner
-$ export BITCOIN_DATA_DIR=$HOME/.bitcoin
+Before executing the script, ensure the following environment variables are defined:
+
+```sh
+export BITCOIND=$HOME/bitcoin/src/bitcoind
+export BITCOIN_CLI=$HOME/bitcoin/src/bitcoin-cli
+export BITCOIN_UTIL=$HOME/bitcoin/src/bitcoin-util
+export MINER=$HOME/bitcoin/contrib/signet/miner
+export BITCOIN_DATA_DIR=$HOME/.bitcoin
 ```
 
 # Blockchain Explorer
+A block explorer is needed to display participants' mined blocks on the custom `signet`. A custom
+`signet` Bitcoin node, [`electrs`](https://github.com/romanz/electrs), and
+[`mempool.space`](https://github.com/mempool/mempool) is used for this purpose.
 
-In order to allow the audience to explore the blockchain, it's also recommended to provide a blockchain explorer.
+> Note: The steps for deploying a local `mempool.space` have not yet been automated and will need to
+be performed manually.
 
-The steps for deploying a local `mempool.space` have not yet been automated, so they need to be performed manually.
+## `electrs`
+Clone, checkout the `v0.10.5` branch, and configure:
 
-First, clone and start `electrs`:
-```
-$ git clone https://github.com/romanz/electrs
-$ cd electrs
-$ git checkout v0.10.5
-$ cat << EOF > electrs.toml
+```sh
+git clone https://github.com/romanz/electrs
+cd electrs
+git checkout v0.10.5
+cat << EOF > electrs.toml
 network="signet"
 auth="mempool:mempool"
 EOF
-$ cargo run -- --signet-magic=54d26fbd
 ```
 
-Then, clone `mempool`:
+Run the server:
+
+```sh
+cargo run -- --signet-magic=54d26fbd
 ```
-$ git clone https://github.com/mempool/mempool
-$ cd mempool
-$ git checkout v2.5.0
+
+## `mempool.space`
+Clone and checkout the `v2.5.0` branch:
+
+```sh
+git clone https://github.com/mempool/mempool
+cd mempool
+git checkout v2.5.0
 ```
 
-We are going to use the docker deployment, so we need to adjust some configs.
-```diff
+The docker deployment is used with the following adjustments to the `docker/docker-compose.yml`:
+
+```sh
+git diff docker/docker-compose.yml
 diff --git a/docker/docker-compose.yml b/docker/docker-compose.yml
-index 68e73a1c8..98fa6a174 100644
+index 68e73a1c8..300aa3d80 100644
```

Comment:
@plebhash take note that I changed:
* `ELECTRUM_HOST: "172.27.0.1"` -> `ELECTRUM_HOST: "host.docker.internal"`
* `ELECTRUM_PORT: "50002"` -> `ELECTRUM_PORT: "60601"`
* `CORE_RPC_HOST: "172.27.0.1"` -> `CORE_RPC_HOST: "host.docker.internal"`

This is how I got it working for me locally. I am not sure how it behaves on the SRI VM as I cannot test because there is no `docker-compose` binary available and I do not have `sudo` rights to install.